src/plugins/intel_npu/src/plugin/npuw/compiled_model.hpp

```diff
@@ -120,9 +120,11 @@ class CompiledModel : public ov::ICompiledModel {
         // w.r.t. function calls
         std::size_t param_base = 0;
         std::vector<ov::Tensor> closure;
+        std::vector<weights::LazyTensor> transformations;
```

Comment:
Abstraction leak here, it is Weight Bank who should track the lazy tensors.

Reply:
Discussed